corehq/apps/locations/permissions.py

```diff
@@ -196,6 +205,8 @@ def is_location_safe(view_fn, view_args, view_kwargs):
         return view_kwargs['resource_name'] in LOCATION_SAFE_TASTYPIE_RESOURCES
     if getattr(view_fn, '_conditionally_location_safe_function', False):
         return view_fn._conditionally_location_safe_function(view_fn, *view_args, **view_kwargs)
+    if getattr(view_fn, 'is_hq_report', False):
+        return view_kwargs['report_slug'] in LOCATION_SAFE_HQ_REPORTS
```

Comment:
remind me why the `is_location_safe` statement above doesn't trigger for HQ reports?